src/Symfony/Component/Translation/Translator.php

```diff
@@ -459,8 +502,13 @@ private function loadFallbackCatalogues($locale)
         }
     }
 
+    /**
+     * @deprecated since version 2.8, to be removed in 3.0. Rely on FallbackTranslatorBag instead.
+     */
     protected function computeFallbackLocales($locale)
     {
+        trigger_error('The '.__METHOD__.' method is deprecated since version 2.8 and will be removed in 3.0. Rely on FallbackTranslatorBag instead.', E_USER_DEPRECATED);
```

Comment:
1. I think so, if someone override public method too he can't get the deprecation notice.
2. It depends, for example to override `computeFallbackLocales` method they must use custom [FallbackTranslatorBag](https://github.com/aitboudad/symfony/blob/translation_loader_resolver/src/Symfony/Component/Translation/FallbackTranslatorBag.php).